certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "description": "Gets or Sets CommitmentNumber",
+          "type": "string"
+        },
+        "binderNumber": {
+          "description": "Gets or Sets BinderNumber",
+          "type": "string"
+        },
+        "ownerInsured": {
+          "description": "Gets or Sets OwnerInsured",
+          "type": "string"
+        },
+        "currentHolders": {
+          "description": "Gets or Sets CurrentHolders",
+          "type": "string"
+        },
+        "interestInLand": {
+          "description": "Gets or Sets InterestInLand",
+          "type": "string"
+        },
+        "legalDescription": {
+          "description": "Gets or Sets LegalDescription",
+          "type": "string"
+        },
+        "lienDescription": {
+          "description": "Gets or Sets LienDescription",
+          "type": "string"
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "format": "date-time",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.SurveyScheduleBItem2Type": {
+      "description": "Gets or Sets SurveyScheduleBItem2Type",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.StandardLanguageListType": {
+      "description": "Gets or Sets StandardLanguageListType",
+      "enum": [
+        "Exceptions",
+        "SubordinateLiens",
+        "RestrictiveCovenants",
+        "RequiredDocuments",
+        "Requirements",
+        "PolicyInsuredMortgage",
+        "PolicyExceptions",
+        "PolicyRestrictiveCovenants",
+        "PolicySubordinateLiens",
+        "PolicyOwnerScheduleB",
+        "Custom",
+        "NotesToCloser",
+        "Unknown"
+      ],
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.AppliesToType": {
+      "description": "Gets or Sets AppliesToType",
+      "enum": [
+        "None",
+        "Mortgage",
+        "Owner",
+        "All"
+      ],
+      "type": "string",
+      "x-nullable": true
+    },
+    "PubCS.TitleOrder.Api.Models.StandardLanguage": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguageListType"
+        },
+        "appliesTo": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.AppliesToType"
+        },
+        "code": {
+          "description": "Gets or Sets Code",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or Sets Description",
+          "type": "string"
+        },
+        "rtf": {
+          "description": "Gets or Sets Rtf",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedB": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.